moviefind/src/js/components/Movie.js

```diff
@@ -1,28 +1,30 @@
-var React =require('react');
+var React = require('react');
 var AppActions = require('../actions/AppActions');
 var AppStore = require('../stores/AppStore');
 
 var Movie = React.createClass({
 
-	render: function(){
+	render: function () {
 		var link = 'https://www.imdb.com/title/' + this.props.movie.imdbID;
-		return( 
-			<div className ="well">
-				<div className ="row">
-					<div className ="col-md-4">
-					<img className="thumbnail" src = {this.props.movie.Poster}/>
+		return (
+			<div className="well">
+				<div className="row">
+					<div className="col-md-4">
+						<img className="thumbnail" src={this.props.movie.Poster} />
 					</div>
 
-				<div className ="col-md-8">
-				<h4>{this.props.movie.Title}</h4>
-				<ul className ="list-group">
-					<li className ="list-group-item"> Year Released : {this.props.movie.Year}</li>
-					<li className ="list-group-item"> IMDB ID: : {this.props.movie.imdbID}</li>
-				</ul> 
-				<a className = "btn btn-primary" href ={link} target ="_blank">View on IMDB </a>
+					<div className="col-md-8">
```

Comment:
Adding more descriptive variable names for `name` and `info` would greatly enhance the clarity and maintainability of the code. For instance, instead of `name`, consider using `genre` or `director` if those are the intended data points.